Library/Formula/sha2.rb

```diff
@@ -0,0 +1,20 @@
+class Sha2 < Formula
+  homepage "http://www.aarongifford.com/computers/sha.html"
+  url "http://www.aarongifford.com/computers/sha2-1.0.1.tgz"
+  sha1 "ffbc69e0b0ab47f70e61aeb14fa16ba2b74dc728"
+
+  option "without-check", "Skip compile-time tests"
+
+  def install
+    system ENV.cc, "-O2", "-DSHA2_UNROLL_TRANSFORM", "-Wall", "-o", "sha2", "sha2prog.c", "sha2.c"
```

Comment:
No ` Makefile `, no. I can draw up a patch and submit it upstream, but upstream activity is limited here.

> Can scrap the -O2

Will do.

> Why the -DSHA2_UNROLL_TRANSFORM

Marginally bigger executable in return for much quicker hashing.

> or -Wall

Shows compiler warnings. Useful to see if something breaks horribly, etc.